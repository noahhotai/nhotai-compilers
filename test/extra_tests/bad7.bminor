// Global arrays not having constant vars
a: array [2] integer = {1, 2+3};

c: integer = 3;
a: array [c] integer = {1, 2, 3};


d: array [2+3] integer;
e: array [4] integer = {c, c+2, 3};
