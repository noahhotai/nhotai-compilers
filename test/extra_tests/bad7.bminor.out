Program parsed successfully.
error: redeclarationc resolves to global c
c resolves to global c
Program failed to resolve.
Program type checked successfully.
